talent/job-descriptions/offers/data-processing.md

```diff
@@ -0,0 +1,30 @@
+Software Engineer in the Data Processing team
+------------------------------------------
+
+### [Location & Seniority](../location-seniority-section.md)
+
+**Compensation:** 49-53,000 Euros + 0.095% [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md) for senior engineers
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+The Data Processing team is developing source{d}'s high-level code analysis applications for running scalable processing pipelines for source code analysis. Written mostly in Go, with some Scala components, it aims to be robust, friendly, flexible and capable of running on large-scale distributed clusters over petabytes of data.
+
+The team projects are:
+
+* [go-mysql-server](https://github.com/src-d/go-mysql-server/): a SQL engine with a MySQL interface, written in Go.
+* [gitbase](https://github.com/src-d/gitbase): a SQL interface to Git repositories, written in Go.
+* [gitbase-spark-connector](https://github.com/src-d/gitbase-spark-connector) gitbase connector for Apache Spark, written in Scala.
+
+You will be expected to have strong backend coding skills in at least two languages and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated, on the other hand Go is not a strict requirement; we strongly believe that it can be learned easily by any skilled developer and care a lot more about our team's mindset and prior experience than any specific skills.
```

Comment:
The contrast of Go with the other skills makes it seem as if its optionality is different from the optional foreknowledge of Scala and Spark. Is it correct to say that knowledge of Scala, Spark, and Go are all desirable but not strict requirements?

If so, then I would propose re-wording this paragraph slightly, maybe along these lines:

> You will be expected to have strong backend coding skills in at least two languages, and very good algorithmic problem-solving skills. Skill at programming in Scala and Go, and knowledge about Apache Spark, are not required but will be highly appreciated. For Go in particular we strongly believe it can be learned by any skilled developer, and we care much more about our team's mindset and prior experience than specific language skills.